x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/checkpoint/DefaultCheckpointProvider.java

```diff
@@ -263,43 +298,62 @@ public void getCheckpointingInfo(long lastCheckpointNumber, TransformIndexerPosi
 
         // <3> got the source checkpoint, notify the user
         ActionListener<Map<String, long[]>> checkpointsByIndexListener = ActionListener.wrap(
-                checkpointsByIndex -> {
-                    checkpointingInfoBuilder.setSourceCheckpoint(
-                        new TransformCheckpoint(transformConfig.getId(), timestamp, -1L, checkpointsByIndex, 0L));
-                    listener.onResponse(checkpointingInfoBuilder.build());
-                },
-                e -> {
-                    logger.debug((Supplier<?>) () -> new ParameterizedMessage(
-                            "Failed to retrieve source checkpoint for transform [{}]", transformConfig.getId()), e);
-                    listener.onFailure(new CheckpointException("Failure during source checkpoint info retrieval", e));
-                }
-            );
+            checkpointsByIndex -> {
+                checkpointingInfoBuilder.setSourceCheckpoint(
+                    new TransformCheckpoint(transformConfig.getId(), timestamp, -1L, checkpointsByIndex, 0L)
+                );
+                listener.onResponse(checkpointingInfoBuilder.build());
+            },
+            e -> {
+                logger.debug(
+                    (Supplier<?>) () -> new ParameterizedMessage(
+                        "Failed to retrieve source checkpoint for transform [{}]",
+                        transformConfig.getId()
+                    ),
+                    e
+                );
+                listener.onFailure(new CheckpointException("Failure during source checkpoint info retrieval", e));
+            }
+        );
 
         // <2> got the next checkpoint, get the source checkpoint
         ActionListener<TransformCheckpoint> nextCheckpointListener = ActionListener.wrap(
-                nextCheckpointObj -> {
-                    checkpointingInfoBuilder.setNextCheckpoint(nextCheckpointObj);
-                    getIndexCheckpoints(checkpointsByIndexListener);
-                },
-                e -> {
-                    logger.debug((Supplier<?>) () -> new ParameterizedMessage(
-                            "Failed to retrieve next checkpoint [{}] for transform [{}]", lastCheckpointNumber + 1,
-                            transformConfig.getId()), e);
-                    listener.onFailure(new CheckpointException("Failure during next checkpoint info retrieval", e));
-                }
-            );
+            nextCheckpointObj -> {
+                checkpointingInfoBuilder.setNextCheckpoint(nextCheckpointObj);
+                getIndexCheckpoints(checkpointsByIndexListener);
+            },
+            e -> {
+                logger.debug(
+                    (Supplier<?>) () -> new ParameterizedMessage(
+                        "Failed to retrieve next checkpoint [{}] for transform [{}]",
```

Comment:
```suggestion
                        "[{}] failed to retrieve next checkpoint [{}]",
```
And flip the parameters. Its much easier to grep and visually understand logs if they are all formatted the same. Putting `[transformID]` at the start of the log unifies the formatting.